addon/templates/components/polaris-choice-list.hbs

```diff
@@ -17,6 +17,12 @@
         checked=choice.isSelected
         onChange=(action "updateSelectedChoices" choice)
       }}
+
+      {{#if choice.extra}}
+        <div class="Polaris-ChoiceList__ChoiceChildren">
+          {{component choice.extra choice=choice}}
```

Comment:
Can we call this something more meaningful and in line with the React implementation than `extra`, maybe `childComponent`? Also, to be in keeping with the React version, this should only pass `choice.isSelected` into the child component.

Reply:
if you think `childComponent` is more inline with react...sure....don't really care about naming here, since either way it won't match react version

Reply:
It's certainly clearer as to its meaning that `extra` so yes - just because it won't ever match the React version exactly doesn't mean it shouldn't be clearly named 😜